master/usage/ip-pool-manipulation.md

```diff
@@ -0,0 +1,82 @@
+---
+title: IP Pool Manipulation
+---
+The 
+[IP Pools]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool)
+that Calico starts up with can be manipulated and changed.  This document
+describes how to do that using the `calicoctl` utility.
+
+## Retrieving IP Pool information
+
+The current IP Pools can be retrieved with the following commmand.
+
+```shell
+calicoctl get ipPool
+```
+
+This will return information similar to the following.
+
+```shell
+CIDR
+192.168.0.0/16
+
+```
+
+To retrieve the information about a specific IP Pool you must specify the pool
+also.  The `-o yaml` will output in a format that can be captured, manipulated,
+and then used to apply the desired changes.
+
+```shell
+calicoctl get ipPool 192.168.0.0/16 -o yaml
+```
+
+## Manipulating an existing pool
+
+### Retrieve current IP Pool config
+```
+calicoctl get ipPool 192.168.0.0/16 -o yaml > pool.yaml
+```
+
+Example of saved `pool.yaml`.
+
+```shell
+- apiVersion: v1
+  kind: ipPool
+  metadata:
+    cidr: 192.168.0.0/16
+  spec: {}
+```
+
+### Modify IP Pool config
+
+Edit the saved `pool.yaml` as desired.  See 
+[IP Pools]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool)
+for the options that can be updated, added, or removed.
+
+Example `pool.yaml` modified to enable `ipip` and `nat-outgoing`.
+
+```shell
+- apiVersion: v1
+  kind: ipPool
+  metadata:
+    cidr: 192.168.0.0/16
+  spec:
+    ipip:
+	  enabled: true
+    nat-outgoing: true
+```
+
+Load the changed file.
+
+```shell
+calicoctl apply -f pool.yaml
+```
+
+## Deleting an existing pool
```

Comment:
Not necessary but I thought it would be good for the case where someone inadvertently changes the pool they capture and then need to delete the newly created one.  I guess that might be an argument for using the replace command instead of apply like I did.